yarn.lock

```diff
@@ -2345,10 +2345,10 @@ yallist@^2.1.2:
   resolved "https://registry.yarnpkg.com/yallist/-/yallist-2.1.2.tgz#1c11f9218f076089a47dd512f93c6699a6a81d52"
   integrity sha1-HBH5IY8HYImkfdUS+TxmmaaoHVI=
 
-yaml-ast-parser-custom-tags@0.0.43:
-  version "0.0.43"
-  resolved "https://registry.yarnpkg.com/yaml-ast-parser-custom-tags/-/yaml-ast-parser-custom-tags-0.0.43.tgz#46968145ce4e24cb03c3312057f0f141b93a7d02"
-  integrity sha512-R5063FF/JSAN6qXCmylwjt9PcDH6M0ExEme/nJBzLspc6FJDmHHIqM7xh2WfEmsTJqClF79A9VkXjkAqmZw9SQ==
+yaml-language-server-parser@next:
+  version "0.1.1-2b9337f.0"
```

Comment:
Is this what you expect, that using `next` will create a version that is higher then the one released?

Reply:
It's the next version on npm: https://www.npmjs.com/package/yaml-language-server-parser/v/0.1.1-2b9337f.0

Reply:
`next` is only version tag, we publish package with some real version, which include hash of the commit, and point `next` to that version.